docs/source/adapters/site.rst

```diff
@@ -415,7 +432,19 @@ Available adapter configuration options
               Walltime: '1440'
               Partition: normal
               StartupCommand: 'pilot_clean.sh'
+              SubmitOptions:
+                short:
+                  C: "intel"
+                long:
+                  gres: "gpu:2,mic:1"
             twelve_hours:
+              Walltime: '360'
+              Partition: normal
+              StartupCommand: 'pilot_clean.sh'
+              SubmitOptions:
+                long:
+                  gres: "gpu:2,mic:1"
+            six_hours:
```

Comment:
I think you accidentally mixed up the six_hours and twelve_hours drone here.
```suggestion
            six_hours:
              Walltime: '360'
              Partition: normal
              StartupCommand: 'pilot_clean.sh'
              SubmitOptions:
                long:
                  gres: "gpu:2,mic:1"
            twelve_hours:
```

Reply:
Ups. Thank you for spotting.